endoreg_db/views/misc/upload_views.py

```
@@ -1,162 +1,158 @@
             upload_job = UploadJob.objects.create(
-                file=uploaded_file, content_type=content_type
+                file=uploaded_file,
+                content_type=content_type
             )
-
+            
             # Start asynchronous processing if Celery is available
             if CELERY_AVAILABLE:
                 try:
                     process_upload_job.delay(str(upload_job.id))
                 except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)